src/PostgREST/Error.hs

```diff
@@ -180,22 +180,25 @@ instance JSON.ToJSON ApiRequestError where
     "hint"    .= ("Try changing '" <> child <> "' to one of the following: " <> relHint rels <> ". Find the desired relationship in the 'details' key." :: Text)]
   toJSON (NoRpc schema procName argumentKeys hasPreferSingleObject contentType isInvPost allProcs overloadedProcs)  =
     let func = schema <> "." <> procName
-        prms = "(" <> T.intercalate ", " argumentKeys <> ")"
-        fmtParams =  if null argumentKeys then " function without parameters" else prms <> " function"
+        prms = T.intercalate ", " argumentKeys
+        prmsMsg = "(" <> prms <> ")"
+        prmsDet = " with parameter(s) " <> prms
```

Comment:
That parenthesis is confusing, the message already uses it for compressing the parameters.
```suggestion
        prmsDet = " with parameters " <> prms
```

Reply:
True, I made it more accurate now.